tests/admin_utils/tests.py

```diff
@@ -260,6 +260,11 @@ def test_callable(obj):
             "--"
         )
 
+        self.assertEqual(
```

Comment:
let's use a single line for this assertion as well as single quotes -- the rest of the file could be cleaned up later